.github/actions/install-dashboards/action.yml

```diff
@@ -54,32 +62,65 @@ runs:
       run: git checkout ${{ steps.osd-version.outputs.osd-version }} || git checkout ${{ steps.osd-version.outputs.osd-x-version }}
       working-directory: ./OpenSearch-Dashboards
       shell: bash
+      
+    - name: Get and Setup OpenSearch-Dashboards Windows
+      if: ${{ runner.os == 'Windows' }}
+      run: |
+        curl -SLO https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/${{ env.OPENSEARCH_VERSION }}/latest/windows/x64/zip/dist/opensearch-dashboards/opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-windows-x64.zip
+        unzip -q opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-windows-x64.zip
+      shell: bash
 
+    - name: Get and Setup OpenSearch-Dashboards Linux
+      if: ${{ runner.os == 'Linux' }}
+      run: |
+        wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/${{ env.OPENSEARCH_VERSION }}/latest/linux/x64/tar/dist/opensearch-dashboards/opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-linux-x64.tar.gz
+        tar -xzf opensearch-dashboards-${{ env.OPENSEARCH_VERSION }}-linux-x64.tar.gz
+      shell: bash
+    
     - id: tool-versions
       run: |
         echo "node_version=$(cat .node-version)" >> $GITHUB_OUTPUT
         echo "yarn_version=$(jq -r '.engines.yarn' package.json)" >> $GITHUB_OUTPUT
       working-directory: OpenSearch-Dashboards
       shell: bash
 
-    - uses: actions/setup-node@v1
+    - name: Setup node
+      uses: actions/setup-node@v1
       with:
-        node-version: ${{ steps.tool-versions.outputs.node_version }}
+        node-version: ${{ env.node_version }}
```

Comment:
if we set from the `env` here, should we do `echo "node_version=$(cat .node-version)" >> $GITHUB_ENV` for the above 2 lines? Instead of `>> $GITHUB_OUTPUT`

Reply:
Yes good catch, done.